logs/logs.go

```diff
@@ -144,9 +145,17 @@ func (l *LogAgent) runSrcToDest(src LogSrc, dest LogDest) {
 	eventsCh := make(chan LogEvent)
 	defer src.Stop()
 
+	var mu sync.Mutex
+	closed := false
 	src.SetOutput(func(e LogEvent) {
+		mu.Lock()
```

Comment:
Having to acquire a lock for every single log event publish seems excessive.

Reply:
removed